.github/workflows/migrations.yml

```diff
@@ -1,99 +1,65 @@
-name: migrations
+name: generate migration sql
```

Comment:
Shouldn't we keep this as "migrations" and move the `check-missing-migration` job from the python workflow to here?

Reply:
They have differing workflow conditions

Reply:
Oh, right. Do we want to be consistent with workflow filename and name then?

Reply:
I'm gonna revert name changes until we come up with a coherent naming strategy